benchmark/README.md

```diff
@@ -19,13 +19,15 @@ $ git clone https://github.com/pingcap/grpc.git
 
 ```
 $ cd grpc-rs
+$ git submodule update --init --recursive
```

Comment:
We have customize step `cargo xtask submodule`. It will only clone needed submodules.

Reply:
I have had an error due to missing dependencies and I had to use git submodule to obtain them

Reply:
Do you mean `cargo xtask submodule` fail? Can you paste the error here?

Reply:
I didn't see in the README that I should use `cargo xtask submodule`

That's why I suggested to use `git submodule update --init --recursive` 
should include  `cargo xtask submodule` in the README

Reply:
Yes, I agree there should be instructions to initialize the submodules in README.md. I just suggest to use `cargo xtask submodule` instead of `it submodule update --init --recursive`.

> should include cargo xtask submodule in the README

That's exact what I mean.